s3torchbenchmarking/src/s3torchbenchmarking/benchmark.py

```diff
@@ -96,7 +121,9 @@ def make_dataset(
                 prefix_uri, region=region, transform=load_sample
             )
             return dataset
-    elif kind == "fsspec":
+    elif kind in ("fsspec", "mountpoint"):
```

Comment:
Should we use [FileLister](https://pytorch.org/data/main/generated/torchdata.datapipes.iter.FileLister.html#torchdata.datapipes.iter.FileLister)/[FileOpener](https://pytorch.org/data/main/generated/torchdata.datapipes.iter.FileOpener.html#torchdata.datapipes.iter.FileOpener) for `mountpoint` or there are no difference in performance with `FSSpecFileLister`/`FSSpecFileOpener` when dealing with files stored localy?

Reply:
This is a good question. I haven't verified the difference so far. We should analyze this separately and pick the more performant implementation to make a fairer analysis(rather than expose this as another knob for users).